spec/models/user_spec.rb

```diff
@@ -1057,6 +1057,24 @@ def provider_username(service_name)
       user.follow(org)
       expect(user.reload.following_orgs_count).to eq(1)
     end
+
+    def create_reaction(user, article)
+      Reaction.create!(
+        user_id: user.id,
+        reactable_id: article.id,
+        reactable_type: "Article",
+        category: "readinglist",
+      )
+    end
```

Comment:
Is there a reason for this method over the classic `create(:reading_reaction)` factory?

Reply:
Nope! That was a lazy copy/paste, updated! :)